images/users/zestyio.svg

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Thanks for your PR. As mentionned at the bottom of the user page (http://babeljs.io/users). Could you provide an optimized 500x200 SVG.